### PR TITLE
Release v0.11.0

### PR DESCRIPTION
**Pull request by nokara26**
This release includes following PR.
#89

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,18 +17,23 @@ Variant translation library for Clojure.
...
+[varity "0.11.0"]
 ```
 
 To use varity with Clojure 1.8, you must include a dependency on
 [clojure-future-spec](https://github.com/tonsky/clojure-future-spec).
 
+## Breaking changes in 0.11.0
+
+We fixed following `varity.vcf-to-hgvs` implementation.
+It is inconvenient to throw an exception in `vcf-variant->protein-hgvs` when variants overlapping a boundary of exon/intron even if `vcf-variant->coding-dna-hgvs` succeed in `vcf-to-hgvs.vcf-variant->hgvs`. So we changed to return nil.
```
Comment (federkasten) [inline README.md:35]:
```suggestion
We fixed the `varity.vcf-to-hgvs` implementation.
It is confusing to throw the exception in `vcf-variant->protein-hgvs` when a variant overlaps the exon-intron boundaries, even if coding DNA HGVS is available. So we changed the behavior to return protein HGVS as `nil`.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
 # Changelog
 
+## [0.11.0] - date TBD
+
+### Fixed
+
+- Return protein-hgvs nil when variants overlapping exon/intron boundaries. [#89](https://github.com/chrovis/varity/pull/89)
```
Comment (federkasten) [inline CHANGELOG.md:7]:
```suggestion
- Return protein HGVS as `nil` if a variant overlaps exon-intron boundaries. [#89](https://github.com/chrovis/varity/pull/89)
```

**Review by federkasten — APPROVED** [review]

**Review by totakke — APPROVED** [review]
LGTM

**Comment by totakke** [comment]
👍 https://clojars.org/varity/versions/0.11.0 🚀

**Pull request merged**